caesars-palace/blocks/header/header.js

```diff
@@ -2,7 +2,9 @@ import { getMetadata, decorateIcons } from '../../scripts/lib-franklin.js';
 
 // media query match that indicates mobile/tablet width
 const isDesktop = window.matchMedia('(min-width: 1170px)');
-// const MAX_NAV_ITEMS_DESKTOP = 6;
+const isLargeDesktop = window.matchMedia('(min-width: 1440px)');
+const MAX_NAV_ITEMS_DESKTOP = 7;
```

Comment:
We could use object to keep configuration/media queries config together, eg. 1 object for desktop, for for large desktop, etc.